config/config_helpers.go

```diff
@@ -42,8 +43,12 @@ func resolveTerragruntInterpolation(str string, include *IncludeConfig, terragru
 // Execute a single Terragrunt helper function and return its value as a string
 func executeTerragruntHelperFunction(functionName string, include *IncludeConfig, terragruntOptions *options.TerragruntOptions) (string, error) {
 	switch functionName {
-	case "find_in_parent_folders": return findInParentFolders(terragruntOptions)
-	case "path_relative_to_include": return pathRelativeToInclude(include, terragruntOptions)
+	case "find_in_parent_folders":
+		path, err := findInParentFolders(terragruntOptions)
+		return cleanPathForHcl( path ), err
+	case "path_relative_to_include":
+		path, err := pathRelativeToInclude(include, terragruntOptions)
+		return cleanPathForHcl( path ), err
```

Comment:
I did this like this to make sure we don't double escape as can happen when path_relative_to_include calls find_in_parent_folders